src/lib/apis/related-products.api.ts

```
@@ -0,0 +1,33 @@
+        const res = await fetch(url, {
+            next: { revalidate: 60 }, // cache for 60 seconds
+            headers: {
+                "Content-Type": "application/json",
+            },
+        });
+
+        const payload = (await res.json()) as RelatedResponse;
+
+        // Handle error responses
+        if (!res.ok) {
+            console.error("Failed to fetch related products:", payload);
```

ESLint

[error] Unexpected console statement.

(no-console)